SPECS/mariadb-connector-c/mariadb-connector-c.spec

```diff
@@ -0,0 +1,476 @@
+# For deep debugging we need to build binaries with extra debug info
+# Enable CMake in-source builds
+#   This is is a workaround for the https://fedoraproject.org/wiki/Changes/CMake_to_do_out-of-source_builds
+#   which reverts the CMake behaviour to before F33
+#   The Change owners offered themselves to help fix the affected packages via ProvenPackager rights.
+#   I'm generally in favor of this change, however when I tried to adapt it, I encountered a number of issues.
+#   That's why I disabled it for now.
+%global __cmake_in_source_build 1
+%bcond_with     debug
+# Enable building and packing of the testsuite
+%bcond_without  testsuite
+Summary:        The MariaDB Native Client library (C driver)
+Name:           mariadb-connector-c
+Version:        3.1.10
+Release:        4%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://mariadb.org/
+Source0:        https://downloads.mariadb.org/interstitial/connector-c-%{version}/%{name}-%{version}-src.tar.gz
+Source2:        my.cnf
+Source3:        client.cnf
+Patch0:         cmake_3.21.4_fix.patch
+BuildRequires:  cmake
+BuildRequires:  gcc-c++
+# auth_gssapi_client plugin
+BuildRequires:  krb5-devel
+# Remote-IO plugin
+BuildRequires:  libcurl-devel
+BuildRequires:  openssl-devel
+BuildRequires:  zlib-devel
+Requires:       %{_sysconfdir}/my.cnf
+# More information: https://mariadb.com/kb/en/mariadb/building-connectorc-from-source/
+%if %{with testsuite}
+Patch1:         testsuite.patch
```

Comment:
Let's always include this to be certain we build consistent SRPMs by simply removing `%bcond_with testsuite` - I doubt we'll need to change that.